CONTRIBUTING.md

```diff
@@ -39,4 +39,32 @@ If you need to make changes to the pull request, simply repeat steps 2-4.  Addin
 
 The majority of code in MonoGame is cross-platform and must build and behave correctly on all supported platforms.  All pull requests to MonoGame are built using an automated system that compiles the pull request on all supported platforms and will report any build errors.
 
-Once a pull request has been accepted, your feature branch can be deleted if desired.
\ No newline at end of file
+Once a pull request has been accepted, your feature branch can be deleted if desired.
+
+Generating Documentation
+========================
+Monogame generates its documentation using the [SharpDoc library] (https://github.com/xoofx/SharpDoc)
+
+You can view the documentation for your changes both locally, and (after you have submitted a PR) on the build server.
+
+### Local documentation generation
+Local documentation generation is not automatic. You have to run the SharpDoc tool to generate it for you.
+The commandline for doing this is: `ThirdParty\Dependencies\SharpDoc\SharpDoc.exe -config <config file> -output <output directory>`
```

Comment:
Put the command line here on a separate line and indented.  It looks better in the final doc.